ts/features/zendesk/screens/ZendeskSeeReportsRouters.tsx

```diff
@@ -89,7 +89,7 @@ const ZendeskSeeReportsRouters = (props: Props) => {
     }
   }, [ticketNumber, dispatch]);
 
-  if (!isStrictSome(ticketNumber) && !pot.isNone(ticketNumber)) {
+  if (pot.isLoading(ticketNumber) || pot.isError(ticketNumber)) {
     return (
       <LoadingErrorComponent
```

Comment:
It seems that this component doesn't provide the back navigation:

<img src="https://user-images.githubusercontent.com/16268789/224751851-aee04217-16f5-48dd-9efb-33cb067cfd66.png" width="150" />

Reply:
[fc70b95](https://github.com/pagopa/io-app/pull/4438/commits/fc70b95b2a117b42a7f239450e483a85312b1ba1) should have fixed can you check back?